src/twister2/builder/build_manager.py

```diff
@@ -137,6 +149,9 @@ def _build(self, builder: BuilderAbstract) -> None:
                 self.update_status(BuildStatus.SKIPPED)
                 logger.info(overflow_exception)
             raise
+        except TwisterBuildFiltrationException:
+            self.update_status(BuildStatus.SKIPPED)
+            raise
```

Comment:
We should log something here.

Reply:
I added log here to be "closer" to source of this filtration/exception:
https://github.com/zephyrproject-rtos/twister/blob/c3b221652be9b4f1654557d01d00390dd19f09ae/src/twister2/builder/build_filter_processor.py#L34-L36